ansible/simple/setup_infrastructure.yml

```diff
@@ -0,0 +1,69 @@
+---
+- hosts: localhost
+  tasks:
+
+  - name: Create new region region1
+    purestorage.fusion.fusion_region:
+      app_id: "{{ ansible_env.API_CLIENT}}"
+      key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+      state: present # or absent
+      name: region1
+      display_name: "region1"
+
+  - name: Create new Availability Zone az1
+    purestorage.fusion.fusion_az:
+      app_id: "{{ ansible_env.API_CLIENT}}"
+      key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+      state: present # or absent
+      name: az1
+      display_name: "az1"
+
+  - name: Create new network interface group interface_group1 in AZ az1
+    purestorage.fusion.fusion_nig:
+      app_id: "{{ ansible_env.API_CLIENT}}"
+      key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+      state: present # or absent
+      name: "interface_group1"
+      availability_zone: "az"
+      region: region1
+      mtu: 1500
+      gateway: 172.17.1.1
+      prefix: 172.17.1.0/24
+
+  - name: Create new Storage Endpoint network in AZ default
+    purestorage.fusion.fusion_se:
+      app_id: "{{ ansible_env.API_CLIENT}}"
+      key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+      state: present # or absent
+      name: default
+      availability_zone: az1
+      mtu: 150
+      gateway: 172.17.1.1
+      addresses: ["172.17.1.2/24", "172.17.1.1/24"]
+      provider_subnets: interface_group1
+
+  - name: Create new array flasharray1
+    purestorage.fusion.fusion_array:
+      app_id: "{{ ansible_env.API_CLIENT}}"
+      key_file: "{{ ansible_env.PRIV_KEY_FILE}}"
+      state: present # or absent
+      name: flasharray1
+      display_name: "flasharray1"
+      az: az1
+      hardware_type: flash-array-x
+      appliance_id: 1187351-242133817-5976825671211737520
+      region: region1
+      host_name: flasharray1
+
+  - name: Create new array flasharray2
```

Comment:
We can nuke this task. Since in simple we just want to show an example of how to use a module, just showing it once above is enough